### PR TITLE
about us data from json, responsive

### PR DESCRIPTION
**Pull request by volodblack**
change page about-us, data from json

### CODE REVIEW

File: src/app/core/services/about-us.service.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { HttpClient } from '@angular/common/http';
+import { AboutUs } from '../interfaces/about-us';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AboutUsService {
+
+  private _url: string = "assets/mockinfo/mock-about-us.json"
```
Comment (TsirohOleh) [inline src/app/core/services/about-us.service.ts]: Be consistent, use only single quotes.
private _url: string = 'assets/mockinfo/mock-about-us.json';

File: src/app/core/services/about-us.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  
+  constructor(private http: HttpClient) { }
+
+  getArticle():Observable<AboutUs[]> {
+    return this.http.get<AboutUs[]>(this._url);
```
Comment (TsirohOleh) [inline src/app/core/services/about-us.service.ts]: Use HttpService created by Oleh.

File: src/app/modules/about/about.component.ts [hunk 3]
```diff
@@ -7,9 +9,16 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AboutComponent implements OnInit {
 
-  constructor() { }
+  private articles: any[] = [];
+  private currentUrl: string;
 
+  constructor(private _aboutUsService: AboutUsService, private _router: Router) {
+    _router.events.subscribe((url: any) => this.currentUrl = url);
```
Comment (TsirohOleh) [inline src/app/modules/about/about.component.ts]: Why do you need currentUrl?

File: src/app/modules/about/about.component.ts [hunk 4]
```diff
@@ -7,9 +9,16 @@ import { Component, OnInit } from '@angular/core';
...
+  }
+  
   ngOnInit() {
+    this._aboutUsService.getArticle()
+      .subscribe(data => this.articles = data);
```
Comment (TsirohOleh) [inline src/app/modules/about/about.component.ts]: Typings

File: src/app/modules/about/about.component.ts [hunk 5]
```diff
@@ -7,9 +9,16 @@ import { Component, OnInit } from '@angular/core';
...
 
-  constructor() { }
+  private articles: any[] = [];
+  private currentUrl: string;
 
+  constructor(private _aboutUsService: AboutUsService, private _router: Router) {
+    _router.events.subscribe((url: any) => this.currentUrl = url);
+  }
+  
   ngOnInit() {
```
Comment (TsirohOleh) [inline src/app/modules/about/about.component.ts]: public ngOnInit(): void {

File: src/app/core/services/about-us.service.ts (hunk 2 above)
Comment (TopOleh) [reply src/app/core/services/about-us.service.ts]: Use firebase service instead.

**Pull request closed without merging**